### PR TITLE
[W5.10][W11-A2] Li JiaYao

### PR DESCRIPTION
**Pull request by JYL123**
Extract out the responsibility of Formatting text for display in to a separate class named Formatter.

### CODE REVIEW

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.ui;
+
+/**
+ * Decoration to ui
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java]: please consider writing a better header comment :stuck_out_tongue:

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -95,28 +85,28 @@ public String getUserCommand() {
...
                 version,
                 MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
                 storageFileInfo,
-                DIVIDER);
+                formatter.getDivider());
     }
 
     public void showGoodbyeMessage() {
-        showToUser(MESSAGE_GOODBYE, DIVIDER, DIVIDER);
+        showToUser(MESSAGE_GOODBYE, formatter.getDivider(), formatter.getDivider());
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/TextUi.java]: You can pass the value to be formatted to the formatter and get the formatted string back here to print.

**Comment by okkhoy** [comment]
@JYL123 
Some comments added. Please ack & close the PR after reading comments.

**Comment by JYL123** [comment]
Thanks for review. Changes are made. PR is closing now.

**Pull request closed without merging**